### PR TITLE
class.md recreated

### PR DESCRIPTION
**Pull request by Cesarcas1958**
### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [ ] All writings are my own.
- [ ] My entry follows the Codecademy Docs style guide.
- [ ] My changes generate no new warnings.
- [ ] I have performed a self-review of my own writing and code.
- [ ] I have checked my entry and corrected any misspellings.
- [ ] I have made corresponding changes to the documentation if needed.
- [ ] I have confirmed my changes are not being pushed from my forked `main` branch.
- [ ] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ ] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/html/concepts/attributes/terms/class/class.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
+---
+Title: 'Classes'
```
Comment (SSwiniarski) [inline content/html/concepts/attributes/terms/class/class.md:2]:
```suggestion
Title: 'class'
```

File: content/html/concepts/attributes/terms/class/class.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+Description: 'Classes are HTML attributes used to select one or more elements for the application of styling or logic.'
```
Comment (SSwiniarski) [inline content/html/concepts/attributes/terms/class/class.md:4]:
```suggestion
Description: 'class is a HTML attribute used to select one or more elements for the application of styling or logic.'
```

File: content/html/concepts/attributes/terms/class/class.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+
+Tags:
+  - 'Attributes'
+  - 'Classes'
+  - 'Style'
```
Comment (SSwiniarski) [inline content/html/concepts/attributes/terms/class/class.md:13]: The end of the metadata is missing. There needs to be a CatalogContent: section and a concluding three dashes: `---`

Also, the extra line breaks in the metadata need to be removed.

The complete Metadata should look like this:
```
---
Title: 'The Title'
Description: 'A brief description'
Subjects:
  - 'A subject name'
  - 'A second subject name'
  - 'An nth subject name'
Tags:
  - 'A tag'
  - 'A second tag'
  - 'An nth tag'
CatalogContent:
  - 'learn-example-course'
  - 'path/example-path'
---
```

You can find entries for CatalogContent: [here](https://github.com/Codecademy/docs/blob/main/documentation/catalog-content.md).

File: content/html/concepts/attributes/terms/class/class.md [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+
+Classes are HTML attributes used to select one or more elements for CSS styling or JavaScript purposes.
```
Comment (SSwiniarski) [inline content/html/concepts/attributes/terms/class/class.md:15]:
```suggestion
**`class`** is a HTML attribute used to select one or more elements for CSS styling or JavaScript purposes.
```

File: content/html/concepts/attributes/terms/class/class.md [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+Inside the stylesheet, a class name is selected with the following syntax:
+
+```css
+.class-1 {
+}
+```
+
+A class selector is a name preceded by a period (`.`).
+
+Below is some markup with a `title`, containing a `header-1` and `P1` and  `P2`. The styling for the `header-1 `, elements with the `P1` and `P1`class name received different styling.
```
Comment (SSwiniarski) [inline content/html/concepts/attributes/terms/class/class.md:30]:
```suggestion
Below is some markup with the classes `header-1` and `P1` and  `P2`. Elements with the `header-1`, `P1` and `P2` class names receive different styling.
```

File: content/html/concepts/attributes/terms/class/class.md [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+    <p class="P1">Pizza Man he delivers is a family-owned business since 1964. Our goal is to make the best pizza and have it delivered to your place.</p>
+    <p class="P2">We have 3 locations so far and growing. </p>
+    </body>
+ 
+</html>
+```
+
+## For JavaScript
+
+The `class` attribute is used to perform JavaScript operations on HTML elements. Through the Document Object Model ("DOM"), there are various JavaScript methods built for selecting elements via their class.
```
Comment (SSwiniarski) [inline content/html/concepts/attributes/terms/class/class.md:72]:
```suggestion
The `class` attribute is used to perform JavaScript operations on HTML elements. Through the Document Object Model ("DOM") there are various JavaScript methods built for selecting elements via their class.
```

File: content/html/concepts/attributes/terms/class/class.md [hunk 7]
```diff
@@ -0,0 +1,86 @@
...
+
+Assuming the markup from the last example is linked to a JavaScript file:
+
+```js
+const innerDiv = document
+  .getElementsByClassName('header-1')
+  console.log(innerDiv)
+
```
Comment (SSwiniarski) [inline content/html/concepts/attributes/terms/class/class.md:80]: Remove the extra line breaks

File: content/html/concepts/attributes/terms/class/class.md [hunk 8]
```diff
@@ -0,0 +1,86 @@
...
+
+```
+
+The code above is getting an element with a class property and prints it to the console
```
Comment (SSwiniarski) [inline content/html/concepts/attributes/terms/class/class.md:84]:
```suggestion
The code above gets an element with a class property and prints it to the console.
```

File: content/html/concepts/attributes/terms/class/class.md [hunk 9]
```diff
@@ -0,0 +1,86 @@
...
+
+recreated on 02/24/2023 again
```
Comment (SSwiniarski) [inline content/html/concepts/attributes/terms/class/class.md:86]:
```suggestion
```
Remove this line.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@Cesarcas1958 I did a first review and left a few comments for you.

**Comment by Cesarcas1958** [comment]
Hi,
I made the changes requested. I added and committed and pushed. Is there
anything more I should do? Is it necessary to make a new pull request? I
did not see the option

**Comment by SSwiniarski** [comment]
@Cesarcas1958 You do not need to make a new PR. All your work should be visible on this PR.

**Comment by SSwiniarski** [comment]
@Cesarcas1958 You shouldn't need to make a new PR, but you need to push changes to the branch you created this PR from. It doesn't seem if you've done that.

File: content/html/concepts/attributes/terms/class/class.md [hunk 10]
```diff
@@ -0,0 +1,88 @@
+---
```
Comment (SSwiniarski) [inline content/html/concepts/attributes/terms/class/class.md:1]: You still need to fix the metadata. You have extra line breaks between sections and you're missing the concluding three dashes: `---`

Again, it should look like this:

```
---
Title: 'The Title'
Description: 'A brief description'
Subjects:
  - 'A subject name'
  - 'A second subject name'
  - 'An nth subject name'
Tags:
  - 'A tag'
  - 'A second tag'
  - 'An nth tag'
CatalogContent:
  - 'learn-example-course'
  - 'path/example-path'
---
```

File: content/html/concepts/attributes/terms/class/class.md [hunk 11]
```diff
@@ -0,0 +1,88 @@
...
+
+The `class` attribute is used to perform JavaScript operations on HTML elements. Through the Document Object Model ("DOM") there are various JavaScript methods built for selecting elements via their class.
+
+Assuming the markup from the last example is linked to a JavaScript file:
+
+```js
+const innerDiv = document
+  .getElementsByClassName('header-1')
+  console.log(innerDiv)
+
```
Comment (SSwiniarski) [inline content/html/concepts/attributes/terms/class/class.md:84]: You still need to remove the extra line breaks in the code block.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@Cesarcas1958 Sorry for the delay. I reviewed this and had two comments about things that still need to be addressed.

Tag me in a comment when you've completed the edits.

**Comment by SSwiniarski** [comment]
@Cesarcas1958 It's been three weeks. Are you able to continue working on this?

**Comment by caupolicandiaz** [comment]
@Cesarcas1958 @SSwiniarski I've made the outstanding edits to keep this entry moving along.

File: content/html/concepts/attributes/terms/class/class.md [hunk 12]
```diff
@@ -0,0 +1,87 @@
...
+      best pizza and have it delivered to your place.
+    </p>
+    <p class="P2">We have 3 locations so far and growing.</p>
+  </body>
+</html>
+```
+
+## For JavaScript
+
+The `class` attribute is used to perform JavaScript operations on HTML elements. Through the Document Object Model ("DOM") there are various JavaScript methods built for selecting elements via their class.
```
Comment (SSwiniarski) [inline content/html/concepts/attributes/terms/class/class.md:78]:
```suggestion
The `class` attribute is used to perform [JavaScript](https://www.codecademy.com/resources/docs/javascript) operations on HTML elements. Through the Document Object Model ("DOM") there are various JavaScript methods built for selecting elements via their class.
```

**Review by SSwiniarski — APPROVED** [review]
@Cesarcas1958 @caupolicandiaz I had one more comment, otherwise good for a second review,

**Review by caupolicandiaz — APPROVED** [review]
@Cesarcas1958 @SSwiniarski I think this looks good to go.

**Comment by SSwiniarski** [comment]
@Cesarcas1958 congrats on the new entry. Live link here: https://www.codecademy.com/resources/docs/html/attributes/class

image

@yangc95, @KTom101 this is another one showing @caupolicandiaz as the contributor, not @Cesarcas1958. Do we know what's causing this?

**Comment by yangc95** [comment]
I think we should first rule out any configuration issues.

If @Cesarcas1958 can double check in the Codecademy profile page that the GitHub handle matches, visibility is set to everyone, and emails for both GitHub/Codecademy match. Then log back in with the GitHub option.

If this issue still persists after double-checking profiles/emails, then we'll check for other potential causes.

**Pull request merged**